lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -234,6 +241,27 @@ function invokeEventListeners(listeners, target, eventImpl) {
 
 const wrappedListener = Symbol("inline event listener wrapper");
 
+/**
+ * Normalize the event listeners options argument in order to get always a valid options object
+ * @param   {Object} options         - user defined options
+ * @param   {Array} defaultBoolKeys  - boolean properties that should belong to the options object
+ * @returns {Object} object containing at least the "defaultBoolKeys"
+ */
+function normalizeEventHandlerOptions(options, defaultBoolKeys) {
+  const returnValue = {};
+
+  if (typeof options === "boolean" || options === null || typeof options !== "object") {
```

Comment:
Yeah, agreed. You can add one to to-upstream if you're feeling ambitious, but definitely not required :)